COPYING

```diff
@@ -53,7 +48,7 @@ copyright notice for the entire Package.
 Package, in accordance with the Copyright Holder's procedures.
 
 "You" and "your" means any person who would like to copy, distribute, or modify
-the Package.
+"the Package.
```

Comment:
Here's a spurious quote, right?

Reply:
Yes, fixed and force pushed!

That extra quote was added by Sublime Text's auto-wrapping feature, which sometimes considers initial symbols as required punctuation (in this case, it must have assumed it was a multiline string being split).